sql/derived_tables/po_instance.sql

```diff
@@ -1,49 +1,77 @@
+-- Create a local table for purchase order line instance. Every po line may have location ID or holding ID or both can be 'null', if both are 'null'
+--then "no source' is present in pol_location_source.
+--Pol_location depends on how the po is created.
+
 DROP TABLE IF EXISTS po_instance;
 
+
 CREATE TABLE po_instance AS
 SELECT
     json_extract_path_text(po_purchase_orders.data, 'poNumber') AS po_number,
+    po_lines.po_line_number AS po_line_number,
     organization_organizations.code AS vendor_code,
     user_users.username AS created_by,
-    po_lines.po_line_number,
     po_purchase_orders.workflow_status AS po_wf_status,

```

Comment:
I'm not sure if the abbreviation "wf" is clear enough. Maybe change to "po_workflow_status" or just "po_status"?